pkg/crc/machine/machine.go

```diff
@@ -212,11 +217,23 @@ func getDriverOptions(machineConfig config.MachineConfig) interface{} {
 	return driver
 }
 
-func setMachineLogging(logging bool) {
-	if !logging {
-		log.SetOutWriter(ioutil.Discard)
-		log.SetErrWriter(ioutil.Discard)
+func setMachineLogging(logs bool) error {
+	if !logs {
+		log.SetDebug(true)
+		logging.CloseLogFile()
+		logfile, err := logging.OpenLogfile()
+		if err != nil {
+			return err
+		}
+		log.SetOutWriter(logfile)
+		log.SetErrWriter(logfile)
 	} else {
 		log.SetDebug(true)
```

Comment:
@praveenkumar So if `--log-level=debug` we'll only log to console and not to the log file?

Reply:
what about the discard statements?

Reply:
We don't need them any more. before if libmachine logging was disabled, they were discarded, now it is logged to the log file `~/.crc/crc.log`, and if `--log-level debug` is passed on the command line, logs are written only to console.

Reply:
Yes so we don't need the discard since we are going to capture everything in the log file.

Reply:
OK :+1: